neonvm/controllers/config.go

```diff
@@ -22,6 +24,10 @@ type ReconcilerConfig struct {
 	// This field is passed to neonvm-runner as the `-qemu-disk-cache-settings` arg, and is directly
 	// used in setting up the VM disks via QEMU's `-drive` flag.
 	QEMUDiskCacheSettings string
+
+	// ReconcileFailureInterval is the interval after which a VM will be marked as failing if it has
+	// not reconciled successfully.
+	ReconcileFailureInterval time.Duration
```

Comment:
`ReconcileFailureInterval` (at least IMO) implies that something happens each interval. Maybe rename to something like `ReconcileFailureWindow` or `ReconcileFailingPendingPeriod` (taken from grafana :smile:) or something?

(maybe even just `FailurePendingPeriod`? these are the _reconciler_ config options after all; I guess "reconcile" may be assumed already...)

WDYT?

Reply:
Oh, TIL, nice!